### ISSUE TITLE
Get an empty 'set()' when I executed 'model.words' command?

### ISSUE THREAD

**Issue by weizier** [salestock/fastText.py#135]
I just got an empty result when I did 'models.words' command although I could see some words in the 'model.vec' file. Could you help, thanks!

**Comment by pliu19** [comment]
Yes, I think I'm facing the same issues. What I can do now is to use gensim to load model.vec. However, in this case, we can't have the vector of unseen words.

**Comment by datalee** [comment]
are you  in the environment of Windows？i face the same issues

**Comment by jcomfort4** [comment]
+1 I am having the same issue on Windows 10.

**Comment by linjian93** [comment]
+1 me too on windows 7. Don't know how to solver it~

**Comment by tiru1930** [comment]
+1 i am using RHEL 6 , after training the fasttext.supervised and trying to print clf.lables i got empty list

**Comment by tiru1930** [comment]
sorry ,problem is with input csv file, i was able get clf.lables  after training

**Comment by enzolupia** [comment]
Same problem, the model return an empty set and I can't do anything with it.
Did anyone solve the problem?

**Comment by davidlenz** [comment]
I am also facing this issue.
